java-checks/src/main/java/org/sonar/java/checks/spring/SpringBeansShouldBeAccessibleCheck.java

```diff
@@ -163,7 +163,7 @@ private static void writeToCache(InputFileScannerContext context, Collection<Str
     }
   }
 
-  private static Optional<List<String>> readFromCache(InputFileScannerContext context) {
+  private static Optional<List<String>>   readFromCache(InputFileScannerContext context) {
```

Comment:
Bad formatting: 3 spaces instead of one.

```suggestion
  private static Optional<List<String>> readFromCache(InputFileScannerContext context) {
```